Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -249,22 +250,23 @@ - (void)saveUserProfile:(UserProfile *)userProfile
                                                     messageSender:self.messageSender
                                                    profileManager:self];
 
-                [[NSNotificationCenter defaultCenter] postNotificationName:kNSNotificationName_LocalProfileDidChange
-                                                                    object:nil
-                                                                  userInfo:nil];
+                [[NSNotificationCenter defaultCenter]
+                    postNotificationNameAsync:kNSNotificationName_LocalProfileDidChange
+                                       object:nil
+                                     userInfo:nil];
             } else {
                 [[NSNotificationCenter defaultCenter]
-                    postNotificationName:kNSNotificationName_OtherUsersProfileWillChange
-                                  object:nil
-                                userInfo:@{
-                                    kNSNotificationKey_ProfileRecipientId : userProfile.recipientId,
-                                }];
+                    postNotificationNameAsync:kNSNotificationName_OtherUsersProfileWillChange
+                                       object:nil
+                                     userInfo:@{
+                                         kNSNotificationKey_ProfileRecipientId : userProfile.recipientId,
+                                     }];
                 [[NSNotificationCenter defaultCenter]
-                    postNotificationName:kNSNotificationName_OtherUsersProfileDidChange
-                                  object:nil
-                                userInfo:@{
-                                    kNSNotificationKey_ProfileRecipientId : userProfile.recipientId,
-                                }];
+                    postNotificationNameAsync:kNSNotificationName_OtherUsersProfileDidChange
```

Comment:
This is used by many views to update to reflect profile changes.  None of them need to be notified synchronously.